3_javascript/javascript2/week2/indexcopy.html

```diff
@@ -0,0 +1,20 @@
+<head>
+    <title>HyfBay</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" href="main.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    
+    <input type="text" id="searchInput" placeholder="Search For Products..." />
+    <input type="number" id="maxPriceInput" placeholder="Enter Your Max Price" />
+    <input type="number" id="highRate" placeholder="Enter Rate" />
```

Comment:
Shouldn't this also have an event listener? :)

Reply:
I fixed all I hope it works now. Sorry I had some vs code-git problems but I tryied to manage it  :)

Reply:
Thank you! :) I will review the rest now